app/FeatureToggle.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Support\Facades\Cache;
+
+class FeatureToggle extends Model {
+    protected $table = 'feature_toggles';
+    protected $primaryKey = 'toggle_name';
+    protected $keyType = 'string';
+    public $incrementing = false;
+
+    public static function isEnabled($toggle_name) {
+        return FeatureToggle::getToggleValue($toggle_name);
+    }
+     
+    public static function toggle($toggle_name) {
+        $toggle = FeatureToggle::find($toggle_name);
+        $toggle_value = FeatureToggle::getToggleValue($toggle_name);
+
+        if ($toggle != null) {
```

Comment:
Here's a better way of checking for a value's nullness:
`if (!is_null($toggle))`

Reply:
I'll fix this - not sure why I did it that way (comparing like that with null sometimes won't work).

Reply:
Resolved and squashed into 4ea15ae